assets/js/components/KeyMetrics/hooks/useDisplayCTAWidget.test.js

```diff
@@ -107,10 +107,14 @@ describe( 'useDisplayCTAWidget hook', () => {
 			)
 		);
 
-		const { result } = await renderHook( () => useDisplayCTAWidget(), {
-			registry,
-		} );
+		const { result, waitForRegistry } = await renderHook(
```

Comment:
Not related to E2E obviously but another instability which failed here that this should fix.

See https://github.com/google/site-kit-wp/actions/runs/12915141223/job/36016312697?pr=10103